bread/bread.py

```diff
@@ -538,6 +532,22 @@ def __init__(self):
             raise ValueError("The 'filterset' setting should be on the BrowseView, not "
                              "the Bread view.")
 
+    def additional_context_data(self):
```

Comment:
Since this is a method, the name should be a verb (`get_additional_context_data` would work). Alternatively, you could make it a property since it takes no args other than self and is read-only.